sdk/network/Azure.Management.Network/CHANGELOG.md

```diff
@@ -2,5 +2,110 @@
+
+This is a Public Preview version, so expect incompatible changes in subsequent releases as we improve the product. To provide feedback, please submit an issue in our [Azure SDK for .NET GitHub repo](https://github.com/Azure/azure-sdk-for-net/issues).
+
+### General New Features
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
+    - HTTP pipeline with custom policies
+    - Better error-handling
+    - Support uniform telemetry across all languages
+
+> NOTE: For more information about unified authentication, please refer to [Azure Identity documentation for .NET](https://docs.microsoft.com//dotnet/api/overview/azure/identity-readme?view=azure-dotnet)
+
+### Migration from Previous Version of Azure Management SDK
+
+#### Package Name
+The package name has been changed from `Microsoft.Azure.Management.Network` to `Azure.Management.Network`
+
+#### Management Client Changes
+
+Example: Create a VNet:
+
+Before upgrade:
+```csharp
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Rest;
+
+var credentials = new TokenCredentials("YOUR ACCESS TOKEN");;
+var networkClient = new NetworkManagementClient(credentials);
+networkClient.SubscriptionId = subscriptionId;
+
+// Create VNet
+var vnet = new VirtualNetwork()
+{
+    Location = location,
+    AddressSpace = new AddressSpace() { AddressPrefixes = new List<string>() { "10.0.0.0/16" } },
+    Subnets = new List<Subnet>()
+    {
+        new Subnet()
+        {
+            Name = "mySubnet",
+            AddressPrefix = "10.0.0.0/24",
+        }
+    },
+};
+
+vnet = await networkClient.VirtualNetworks
+    .BeginCreateOrUpdateAsync(resourceGroup, vmName + "_vent", vnet);
+```
+
+After upgrade:
+```csharp
+using Azure.Identity;
+using Azure.Management.Network;
+using Azure.Management.Network.Models;
+
+var networkClient = new NetworkManagementClient(subscriptionId, new DefaultAzureCredential());
+var virtualNetworksClient = networkClient.GetVirtualNetworksClient();
+
+// Create VNet
+var vnet = new VirtualNetwork()
+{
+    Location = location,
+    AddressSpace = new AddressSpace() { AddressPrefixes = new List<string>() { "10.0.0.0/16" } },
+    Subnets = new List<Subnet>()
+    {
+        new Subnet()
+        {
+            Name = "mySubnet",
+            AddressPrefix = "10.0.0.0/24",
+        }
+    },
+};
+
+var response = await virtualNetworksClient.StartCreateOrUpdateAsync(resourceGroup, vmName + "_vent", vnet);
+vnet = await response.WaitForCompletionAsync();
+```
+
+#### Object Model Changes
+
+Example: Create a VirtualNetworkGatewayConnection Model
+
+Before upgrade:
+```csharp
+var virtualNetworkGatewayConnection = new VirtualNetworkGatewayConnection()
```

Comment:
is this a good example? should we use an example that does not have so much dependency on other responses?